docs/getting-started.md

```diff
@@ -10,7 +10,7 @@ We’ll use the following repositories:
 # Check the system requirements
 
 !!! info
-    Be aware that udata now requires Python **>3.7,<3.10** to work. 
+    Be aware that udata now requires Python **>3.7,<=3.11** to work. 
```

Comment:
I don't think we support 3.7 anymore :-) I think it's at least 3.9

Reply:
Great, updated.